tests/term_property_json_shape.rs

```diff
@@ -0,0 +1,781 @@
+use nanobot::ldtab::{
+    get_label_map, get_predicate_map_hiccup, get_prefix_map, get_property_map, get_subject_map,
+};
+use nanobot::sql::{parse, select_to_sql, select_to_url, Direction, Operator, Select};
+use serde_json::json;
+use sqlx::sqlite::{SqlitePool, SqlitePoolOptions};
+use std::collections::HashSet;
+
+#[tokio::test]
+async fn test_get_prefix_map() {
+    let connection = "src/resources/test_data/zfa_excerpt.db";
+    let connection_string = format!("sqlite://{}?mode=rwc", connection);
+    let pool: SqlitePool = SqlitePoolOptions::new()
+        .max_connections(5)
+        .connect(&connection_string)
+        .await
+        .unwrap();
+
+    let mut curies = HashSet::new();
+    curies.insert(String::from("obo:ZFA_0000354"));
+    curies.insert(String::from("rdfs:label"));
+    let prefix_map = get_prefix_map(&curies, &pool).await.unwrap();
+    let expected_prefix_map = json!({"@prefixes":{"obo":"http://purl.obolibrary.org/obo/","rdfs":"http://www.w3.org/2000/01/rdf-schema#"}});
+    assert_eq!(prefix_map, expected_prefix_map);
+}
+
+#[tokio::test]
+async fn test_get_label_map() {
+    let connection = "src/resources/test_data/zfa_excerpt.db";
+    let connection_string = format!("sqlite://{}?mode=rwc", connection);
+    let pool: SqlitePool = SqlitePoolOptions::new()
+        .max_connections(5)
+        .connect(&connection_string)
+        .await
+        .unwrap();
+
+    let table = "statement";
+
+    let mut curies = HashSet::new();
+    curies.insert(String::from("obo:ZFA_0000354"));
+    let label_map = get_label_map(&curies, &table, &pool).await.unwrap();
+    let expected_label_map = json!({"@labels":{"obo:ZFA_0000354":"gill"}});
+    assert_eq!(label_map, expected_label_map);
+}
+
+#[tokio::test]
+async fn test_get_property_map() {
+    let connection = "src/resources/test_data/zfa_excerpt.db";
+    let connection_string = format!("sqlite://{}?mode=rwc", connection);
+    let pool: SqlitePool = SqlitePoolOptions::new()
+        .max_connections(5)
+        .connect(&connection_string)
+        .await
+        .unwrap();
+
+    let subject = "obo:ZFA_0000354";
+    let table = "statement";
+
+    let property_map = get_property_map(&subject, &table, &pool).await.unwrap();
+    let expected_property_map = json!(
+    {
+      "obo:IAO_0000115": [
+        {
+          "object": "Compound organ that consists of gill filaments, gill lamellae, gill rakers and pharyngeal arches 3-7. The gills are responsible for primary gas exchange between the blood and the surrounding water.",
+          "datatype": "xsd:string",
+          "annotation": {
+            "<http://www.geneontology.org/formats/oboInOwl#hasDbXref>": [
```

Comment:
Why is this an IRI and not a CURIE? I see the `oboInOwl` prefix in the `prefix` table of the SQLite database.

Reply:
I expect that the `oboInOwl` prefix wasn't actually used by LDTab when the SQLite file was created. I pushed a commit that fixes this.

Reply:
That's right. The `oboInOwl` prefix wasn't configured when the SQLite file was created. We probably should have dedicated tests for both IRIs and CURIEs. Even though they are both simply treated as strings in the code and should give rise to the same behaviour, this is not always the case. The function `get_prefixes` [here](https://github.com/ontodev/nanobot.rs/blob/term-properties-json/src/ldtab.rs#L53) is an example. So far, I didn't encounter any issues with this. But the angle brackets used in IRIs might cause issues in the HMTL encoding.